icu_benchmarks/models/alarm_metrics.py

```
@@ -0,0 +1,72 @@
+import numpy as np
+from numpy import ndarray
```

Ruff (0.8.2)

`numpy.ndarray` imported but unused

Remove unused import: `numpy.ndarray`

(F401)